devai-cli/src/devai/util/constants.py

```diff
@@ -0,0 +1,4 @@
+"""Constants used throughout the DevAI CLI."""
```

Comment:
can we delete this `constants.py` file?
 - commands are using 
https://github.com/GoogleCloudPlatform/genai-for-developers/blob/main/devai-cli/src/devai/commands/constants.py#L2